manifests/f/filips/FirefoxPWA/2.4.1/filips.FirefoxPWA.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: filips.FirefoxPWA
+PackageVersion: 2.4.1
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+Installers:

```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
Installers:
```

Reply:
The package installs, but needs the `VCRedist` package to run: 
![image](https://user-images.githubusercontent.com/114952565/217451575-c2dd7ed0-0531-4183-9fdd-315a7a5e62b1.png)

Reply:
Will this install both x64 and x86 versions at the same time? If so, is it possible to only install VCRedist for architecture that is used?

Reply:
> Will this install both x64 and x86 versions at the same time? If so, is it possible to only install VCRedist for architecture that is used?

You would need to have a separate dependencies section for each installer